### PR TITLE
Increase session token expiration

### PR DESCRIPTION
**Pull request by polanjir**
* Increase session token expiration for developers/Testservers
* Add better logging
* Upgrade libs because of dependabot warnings

Fully tested on localhost with no issue.
```
# Start server
gunicorn --reload -b 127.0.0.1:8000 main:app

# Retrive token
curl -k -H "Authorization: Bearer <my-github-token>"  127.0.0.1:8000/token\?role\=cz_dev_developer
```

[DO-1263](https://twisto.myjetbrains.com/youtrack/issue/DO-1263) Resolve TS token expiration issues

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -56,10 +62,12 @@ def exchange_token():
     role = request.args.get('role')
     role_mapping = config['roleMapping'].get(role)
     if not role_mapping:
+        app.logger.debug({'message': 'Unknown role', 'role': role_mapping})
```
Comment (patakovamichaela) [inline main.py:65]: I think that this should be error level logging

Comment (patakovamichaela) [reply main.py:65]: Generally all 400 messages should be logged as error.

**Review by patakovamichaela — CHANGES_REQUESTED** [review]

File: main.py (hunk 1 above)
Comment (polanjir) [reply main.py]: Yes, you're absolutely right. It should be fixed by now 😉

**Review by patakovamichaela — APPROVED** [review]
LGTM

**Pull request merged**